### DISCUSSION TITLE
Should there be a new Termux variant on Play Store?

### DISCUSSION THREAD

**Discussion by willtejeda** [termux/termux-app#3697] [Polls]
I'm curious , with things like UserLAnd being built on top of Termux, if it would be possible (or even globally wanted) to have Termux on the app store under a new name or new name variant (ie: TermuxV2 or Terminator lol ) etc etc 

I've read that the original dev abandoned the project and with them the account for play store was lost, but wouldn't it be possible to create a new account under this new name ?

**Comment by twaik** [comment]
**Termux on Google Play Store is OUTDATED** 
Termux and its plugins are no longer updated on the Google Play Store. This is due to API requirements that introduce new restrictions, which break Termux's basic functionality.
Please switch to the F-Droid version!
Direct link: https://f-droid.org/en/packages/com.termux
**Before** you install it, You **must** __fully uninstall__ current Termux and all plugins. If you don't, you will get a cryptic `app not installed` error!
(that means uninstalling the main app, as well as Termux:API, :Styling, :Widget, :Task, :Float, :Boot)
If you need help with backing up your files:
<https://wiki.termux.com/wiki/Backing_up_Termux>

**Comment by twaik** [comment]
> I've read that the original dev abandoned the project and with them the account for play store was lost

He is back, but publishing termuxon github is not possible due to https://github.com/termux/termux-app/issues/2155

**Comment by willtejeda** [comment]
I find it hard to believe that there's not some workaround for this (especially considering the UserLAnd example )

**Reply by twaik** [reply]
No, there is no workaround. It is against Google Play's TOS.

**Comment by sylirre** [comment]
Such topics don't contribute to Termux development and rather spread FUD in our community. Therefore closed & locked down.

Termux will be on Play Store again when this will be possible (if ever). There some work is being done ([termux-exec](https://github.com/termux/termux-exec/pull/24) and [termux-app google-play branch](https://github.com/termux/termux-app/tree/google-play)) but no guarantee that it would be submitted.
